src/Juvix/Compiler/Builtins/Control.hs

```diff
@@ -0,0 +1,27 @@
+module Juvix.Compiler.Builtins.Control where
+
+import Juvix.Compiler.Abstract.Extra
+import Juvix.Compiler.Builtins.Effect
+import Juvix.Prelude
+
+registerSeq :: (Members '[Builtins, NameIdGen] r) => FunctionDef -> Sem r ()
+registerSeq f = do
+  let u = ExpressionUniverse (Universe {_universeLevel = Nothing, _universeLoc = error "Universe with no location"})
```

Comment:
Maybe we should have the following in `Juvix.Data.Universe`
```
smallUniverseNoLoc :: Universe
smallUniverseNoLoc = smallUniverse (error "Universe with no location")
```
At some point we'll need to figure out how to handle these non-existing locations

Reply:
done